pom.xml

```diff
@@ -49,9 +49,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.webjars</groupId>
+            <groupId>org.webjars.bower</groupId>
```

Comment:
I think I'd rather keep dependencies within the same system of WebJars.  Have you tried to create a Bower WebJar for this artifact?